src/App.js

```diff
@@ -1,24 +1,22 @@
-import logo from './logo.svg';
-import './App.css';
+import React from 'react';
+// import { useDispatch } from 'react-redux';
```

Comment:
- [OPTIONAL] Kindly remove commented code from you code base as this makes your code dirty and less readable.

Reply:
I've removed the commented code as you asked! Thank you for reviewing and approving my project. Have a nice weekend!